### ISSUE TITLE
Show font path

### ISSUE THREAD

**Issue by fakefred** [sdushantha/fontpreview#18]
I sometimes make projects that need fonts bundled. This requires me to find out where I installed my fonts to. The system font viewer isn't really helpful at this. So, I would really appreciate if the path of fonts was displayed after the fontnames given a flag such as `--show-path`.

**Comment by sdushantha** [comment]
I think this is something we could add by having a keybinding. I know @aeghn added some keybindings in #13, so I guess we could have a binding to grab the font path.

**Comment by aeghn** [comment]
What about this?
![p-200210_013237](https://user-images.githubusercontent.com/51843252/74106885-9170d780-4ba5-11ea-9e60-7e0547908fe5.png)

**Comment by sdushantha** [comment]
Yes!!! That is a great idea! Could you change the binding to `Ctrl-i`, which stands for **info**?

**Comment by aeghn** [comment]
Yeah, we can change all the bindings.

And I'll add a help menu tomorrow.
